### PR TITLE
#1813 Updated ADDR WRITE FUNCT- to handle for either 1 or 2 street address fields in a dialog

### PR DESCRIPTION
**Pull request by MeganGeissler**
Issue with recent update to Access_Addder "WRITE" portion. I had adjusted the script to use resi_street_full and mail_steet_full when writing however this didn't handle for the fact that dialogs sometimes have 1 or 2 fields to capture the street address. 

Updated handling to handle for both scenarios. Extra testing on this would be appreciated :). Return Mail and LTC 5181. Use had reported issue with 5181 not writing correct street address which is brought this to our attention.

### CODE REVIEW

**Review by C-Love — APPROVED** [review]
Hold on - There is an issue with the word count in the lines

File: MASTER FUNCTIONS LIBRARY.vbs [hunk 1]
```diff
@@ -1715,26 +1716,52 @@ function access_ADDR_panel(access_type, notes_on_address, resi_line_one, resi_li
...
-	        End If
+			If resi_line_one = "" AND resi_line_two = "" Then 
+				If len(resi_street_full) > 22 Then 
+					resi_words = split(resi_street_full, " ")
+					write_resi_line_one = ""
+					write_resi_line_two = ""
+					For each word in resi_words
+						If write_resi_line_one = "" Then
+							write_resi_line_one = word
+						ElseIf len(write_resi_line_one & " " & word) =< 22 Then
```
Comment (C-Love) [inline MASTER FUNCTIONS LIBRARY.vbs:1727]: If a smaller word is after a longer word - the order or words will get messed up. 

'1135 Fifth Avenue Apartment 1905' became 

'1135 Fifth Avenue 1905'
'Apartment'

We need a boolean to identify if we are moving to line 2

File: MASTER FUNCTIONS LIBRARY.vbs [hunk 2]
```diff
@@ -1715,26 +1716,52 @@ function access_ADDR_panel(access_type, notes_on_address, resi_line_one, resi_li
...
+			Else
+				resi_street_full = trim(trim(resi_line_one) & " " & trim(resi_line_two))
+				If len(resi_line_one) or len(resi_line_two) > 22 Then 
+					resi_words = split(resi_street_full, " ")
+					write_resi_line_one = ""
+					write_resi_line_two = ""
+					For each word in resi_words
+						If write_resi_line_one = "" Then
+							write_resi_line_one = word
+						ElseIf len(write_resi_line_one & " " & word) =< 22 Then
```
Comment (C-Love) [inline MASTER FUNCTIONS LIBRARY.vbs:1750]: This would create the same issue

**Review by C-Love — CHANGES_REQUESTED** [review]
The update worked but we have the possibility of messing up the word order.

**Review by courtrightd — APPROVED** [review]
Testing good!

File: MASTER FUNCTIONS LIBRARY.vbs (hunk 1 above)
Comment (MeganGeissler) [reply MASTER FUNCTIONS LIBRARY.vbs]: @C-Love - I've added boolean for resi and mail handling. Take a peek and let me know what you think!

**Review by C-Love — APPROVED** [review]
Worked great. Logic is solid. I approve.

**Pull request merged**